sec-scanners-config.yaml

```diff
@@ -0,0 +1,8 @@
+module-name: infrastructure-manager
+protecode:
+  - europe-docker.pkg.dev/kyma-project/prod/infrastructure-manager:0.0.1
```

Comment:
I'm not sure about if this version is correct. There is no `0.0.1` tag in the repository

Reply:
I've changed the image in b653984cb16ff3328df7c725c73e8af28fe2ba10